content/about.html

```diff
@@ -33,10 +33,10 @@ <h3>What is OpenRA?</h3>
 
 <div class="about-modblock">
   <div class="about-modimage about-right">
-    <img width="312" height="175" src="/images/about/ingame-cnc.png" />
+    <img width="312" height="175" src="/images/ingame-cnc.png" />
   </div>
   <div class="about-modtext about-left" style="margin-top:10px">
-    <h3><img src="/images/about/mod-cnc-logo.png" width="32" height="32" />Tiberian Dawn</h3>
+    <h3><img src="/images/icons/cnc_64x64.png" width="32" height="32" />Tiberian Dawn</h3>
```

Comment:
Why are these called _64x64?  #15185 references filenames with _32x32.

Reply:
The about page uses the 64px versions to improve visuals on 2x HiDPI displays.  These are separate assets to the new 32px versions which are used by #15185 but not the website.